pkgs/by-name/as/astroterm/package.nix

```
@@ -0,0 +1,72 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  fetchFromGitHub,
+  xxd,
+  meson,
+  ninja,
+  ncurses,
+  argtable,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "astroterm";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "da-luce";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "1zgsyh0igw351zv1bd80i8bx93k4lbhvfb26rs8pzjkbpj8b89rm";
```

```suggestion
    hash = "";
```
Use the newer `hash` pattern, instead of `sha256`. Leave the string blank, and build the package. Nix should then give you a new hash that begins with `sha256-`. Copy that hash here.

Ditto for the `fetchurl` below.